ansible/README.md

```diff
@@ -125,7 +116,7 @@ ansible-playbook -i environments/<environment> setup.yml
 ```
 
 #### Install Prerequisites
-This step needs to be done only once per target environment (in a local setup the development environment and the target environment are the same). It will install necessary prerequisites on all target hosts in the environment.
+This step needs to be done only once per target environment. It will install necessary prerequisites on all target hosts in the environment. No required for local environments since all prerequisites are already installed.
```

Comment:
`No required for local environments since all prerequisites are already installed.` ->
`This step is not required for local environments since all prerequisites are already installed, and therefore may be skipped.`